### DISCUSSION TITLE
Error while installing

### DISCUSSION THREAD

**Discussion by cornel619** [Security-Onion-Solutions/securityonion#11549] [Unsupported Versions]
I keep getting this error when installing. I tried both the Static IP option and DHCP option. What could be the problem? My networking knowledge isn't all that great. I'm using VMware workstation 17
![onion](https://github.com/Security-Onion-Solutions/securityonion/assets/99259716/d6c9243f-71cd-4479-8687-05389788ce46)

**Comment by cornel619** [comment]
I'm getting this. 
![onion2](https://github.com/Security-Onion-Solutions/securityonion/assets/99259716/7f8e9b9a-615f-4665-afd7-225731e502ed)

**Reply by reyesj2** [reply]
Looks like something went wrong during your install of the ISO (It says 'Read-only file system). Did you verify the ISO before starting the install? Following the steps at https://github.com/Security-Onion-Solutions/securityonion/blob/2.4/main/DOWNLOAD_AND_VERIFY_ISO.md
